plugin.video.crossroads/addon.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.crossroads"
+version="1.0.0"
+name="Crossroads Church"
+provider-name="CRDS">
+<requires>
+  <import addon="xbmc.python" version="2.24.0"/>
```

Comment:
python api version should be 2.25.0 to match krypton (the repo to which you are submitting)